homeassistant/components/voip/devices.py

```diff
@@ -21,6 +30,7 @@ class VoIPDevice:
     device_id: str
     is_active: bool = False
     update_listeners: list[Callable[[VoIPDevice], None]] = field(default_factory=list)
+    vad_sensitivity: VadSensitivity = VadSensitivity.DEFAULT
```

Comment:
Instead of storing it on `VoIPDevice`, I suggest that we make it a select entity provided by `assist_pipeline/select.py`, just like we do for the selected pipeline. That way there is a single consistent place that we control this for all integrations that use VAD.